### ISSUE TITLE
Exception java.lang.UnsatisfiedLinkError: dlopen failed: cannot locate symbol "localeconv"

### ISSUE THREAD

**Issue by vtrrsl** [realm/realm-js#1522]
## Expected Results
Have had a new crash reported since updating from version 2.0.6 to 2.0.10

## Actual Results

```
Exception java.lang.UnsatisfiedLinkError: dlopen failed: cannot locate symbol "localeconv" referenced by "[REDACTED_DOMAIN_NAME]"...
java.lang.Runtime.load (Runtime.java:333)
java.lang.System.load (System.java:519)
com.facebook.soloader.DirectorySoSource.loadLibraryFrom (DirectorySoSource.java:71)
com.facebook.soloader.DirectorySoSource.loadLibrary (DirectorySoSource.java:42)
com.facebook.soloader.SoLoader.loadLibraryBySoName (SoLoader.java:299)
com.facebook.soloader.SoLoader.loadLibrary (SoLoader.java:247)
io.realm.react.RealmReactModule.<clinit> (RealmReactModule.java:43)
io.realm.react.RealmReactPackage.createNativeModules (RealmReactPackage.java:15)
com.facebook.react.NativeModuleRegistryBuilder.processPackage (NativeModuleRegistryBuilder.java:106)
com.facebook.react.ReactInstanceManager.processPackage (ReactInstanceManager.java:1191)
com.facebook.react.ReactInstanceManager.processPackages (ReactInstanceManager.java:1161)
com.facebook.react.ReactInstanceManager.createReactContext (ReactInstanceManager.java:1094)
com.facebook.react.ReactInstanceManager.access$800 (ReactInstanceManager.java:112)
com.facebook.react.ReactInstanceManager$4.run (ReactInstanceManager.java:922)
java.lang.Thread.run (Thread.java:864)
```

## Version of Realm and Tooling

- Realm JS SDK Version: 2.0.10
- Node or React Native: React Native
- Client OS & Version:  Android API: 17 - 19, Android OS: 4.4.2 - 4.4.4 
- Which debugger for React Native: None

**Comment by kneth** [comment]
I believe that `localeconv` isn't support for Android API < 21 but React Native support Android API >= 16. So it's an oversight from our side (aka bug!).

Thanks for reporting.

**Comment by vtrrsl** [comment]
No problem, thanks for looking into it :)

**Comment by devbriguy** [comment]
HI @vtrrsl I am testing 2.0.11 and I am seeing the issue still. Were you able to test out realm@2.0.11?

**Comment by vtrrsl** [comment]
@devbriguy Yeah, there haven't been a new release with the fix yet. I reverted back to 2.0.6 and do not have the issue there.

**Comment by kneth** [comment]
@devbriguy @vtrrsl I will probably do a release today.

**Comment by kneth** [comment]
@devbriguy @vtrrsl I have released 2.0.12 with a fix for this.

**Comment by vtrrsl** [comment]
Awesome!
